app/scripts/controllers/tools.js

```diff
@@ -566,12 +566,16 @@ angular.module('oncokbApp')
                 _.each($scope.reviewedData[$scope.data.evidenceType].body, function(item) {
                     tempArr = [];
                     _.each($scope.reviewedData[$scope.data.evidenceType].keys, function(key) {
-                        tempArr.push(item[key]);
+                        if (key === 'citations') {
+                            tempArr.push('=HYPERLINK(\"https://www.ncbi.nlm.nih.gov/pubmed/' + item[key] + '\", \"'+item[key] + '\")');
```

Comment:
item[key] duplicated twice? And shouldn't it be constructed based on number of the pmids in the value?

Reply:
The first item[key] used in link and the second one used as link name.

<img width="610" alt="Screen Shot 2019-08-12 at 2 24 54 PM" src="https://user-images.githubusercontent.com/14971266/62888258-e1610580-bd0c-11e9-9520-378b6ab76d52.png">